app/kubernetes/views/configure/configureController.js

```diff
@@ -18,6 +18,7 @@ class KubernetesConfigureController {
     this.EndpointService = EndpointService;
     this.EndpointProvider = EndpointProvider;
     this.ModalService = ModalService;
+    this.KubernetesMetricsService = KubernetesMetricsService;
 
```

Comment:
do we need to bind the enableMetricsServer function?

Reply:
It is not an async function so no (see comment above)